lib/net/ldap/error.rb

```diff
@@ -0,0 +1,31 @@
+class Net::LDAP
+  class Error < StandardError; end
```

Comment:
:+1: for switching the name from `LdapError` to just `Error`. Does anyone have strong opinions on having a backwards compatibility placeholder?

```ruby
class LdapError < Error
  def message
    "Deprecation warning: Net::LDAP::LdapError is no longer used. Use Net::LDAP::Error or rescue one of it's subclasses. [link to this file]\n" + super
  end
end
```

We're pre 1.0, there's no obligation for us to do this. I'm fine with either because it's an easy error to fix if anyone runs into it and we can document it in History.md